components/BottomAppBar/src/MDCBottomAppBarViewController.m

```diff
@@ -0,0 +1,149 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomAppBarViewController.h"
+
+#import "MDCBottomAppBarAttributes.h"
+#import "private/MDCBottomAppBarView.h"
+
+@interface MDCBottomAppBarViewController ()
+
+@property(nonatomic, strong) MDCBottomAppBarView *bottomBarView;
+
+@end
+
+@implementation MDCBottomAppBarViewController
+
+- (instancetype)initWithViewController:(UIViewController *)viewController {
+  self = [super initWithNibName:nil bundle:nil];
+  if (self) {
+    [self commonMDCBottomAppBarViewControllerInit];
+    [self setViewController:viewController];
```

Comment:
Since this setter isn't overridden, go for the `_`.

Reply:
Will using _ still call the necessary set up code for inserting the view controller in the parent view?

Reply:
Nope! I missed that. Keep as is.